go/vt/vtorc/inst/instance_dao.go

```diff
@@ -200,6 +201,9 @@ func ReadTopologyInstanceBufferable(tabletAlias string, latency *stopwatch.Named
 
 	fs, err = fullStatus(tabletAlias)
 	if err != nil {
+		if config.Config.EnableStalledDiskPrimaryAnalysis && strings.Contains(err.Error(), "stalled disk") {
+			stalledDisk = true
+		}
 		goto Cleanup
```

Comment:
If we don't send the stalled disk as an error back as suggested ☝️, we can set the value in the normal flow below and it would be written as part of `mkInsertOdkuInstances`, and we won't need to change `UpdateInstanceLastChecked`.